docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -64,15 +64,49 @@ If you are reading this topic with the questions listed below, click them to jum
 
 4. Connect to Nebula Graph.
 
+    4.1 Start a Nebula Graph service in a Docker container
```

Comment:
Use the ordered list in standard markdown with correct indentation.

Example:
![image](https://user-images.githubusercontent.com/69501902/106238118-3cf1bc80-623b-11eb-88bd-ddd666101016.png)

It will be transformed to:

4. xxx
    1. sub1xxx
    2. xxxx

5. xxx